examples/nlp/training/trainer.py

```diff
@@ -0,0 +1,143 @@
+﻿#  Copyright (c) maiot GmbH 2021. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+
+import os
+from typing import List, Text, Dict
+
+import tensorflow as tf
+from transformers import TFDistilBertForSequenceClassification
+
+from zenml.core.steps.trainer.tensorflow_trainers.tf_base_trainer import \
+    TFBaseTrainerStep
+from zenml.utils.post_training.post_training_utils import \
+    get_feature_spec_from_schema
+
+
+class UrduTrainer(TFBaseTrainerStep):
+    def __init__(self,
```

Comment:
Do we want to keep this style or have one newline after the bracket? I think we are inconsistent throughout the project but can make a decision now about it

Reply:
I think the newline is redundant if there is no additional info (docstring etc) to put there, personally I think the spacing creates an impression of the code dangling from the class, which strikes me as weird when reading with the flow